e2e_correctness/map_test/test_from_nodes_1/input.cyp

```diff
@@ -0,0 +1,3 @@
+CREATE (Oppenheimer:Movie {id:1, title:'Oppenheimer', released:2023, tagline:'Now I am become Death, the destroyer of worlds.'})
+CREATE (Barbie:Movie {id:2, title:"Barbie", released:2023, tagline:'I am a Barbie girl in the Barbie world.'})
+CREATE (Shawshank:Movie {id:3, title:'The Shawshank Redemption', released:1994, tagline:'Over the course of several years, two convicts form a friendship, seeking consolation and, eventually, redemption through basic compassion.'});
```

Comment:
These tests only test graph structure, unfortunately. They don't test output. We need to use for these ones regular e2e tests.

Reply:
That is if I am not wrong as it seems from the code you are only returning results not updating anything directly

Reply:
I think its not possible to create tests for this procedure then, will explain on daily.